osu.Game/Screens/OnlinePlay/Playlists/PlaylistsMatchSettingsOverlay.cs

```diff
@@ -134,36 +132,21 @@ private void load(OsuColour colours)
                                                                 }
                                                             }
                                                         },
-                                                        new Section("Room visibility")
+                                                        new Section("Allowed attempts (across all playlist items)")
```

Comment:
This is apparently [not how it works](https://github.com/ppy/osu-web/issues/7148#issuecomment-772432047). I verified against the dev instance and it does indeed seem to be

```suggestion
                                                        new Section("Allowed attempts (for each playlist item)")
```

Reply:
The behaviour is correct with the previous wording; we'll get that fixed on osu-web